content/en/docs/Features/multi-cluster.md

```diff
@@ -1,24 +1,56 @@
 ---
-title: "Multi-cluster Deployment"
+title: "Multi-cluster"
 description: "Advanced Mesh Deployment and Multi-cluster support."
 ---
 
-A basic Istio mesh deployment has a single control plane with a single data plane, deployed on a single Kubernetes cluster. But Istio supports a variety of advanced 
+{{% alert color="warning" %}}
+Support for multi-cluster deployments is currently **experimental** and subject to change. It is not suitable for production environments.
```

Comment:
I'm good with keeping the "experimental" label but removing the production warning. The only "dangerous" thing I see is in giving Kiali remote access to clusters. However this can be read only and should be gated by RBAC if Kiali is not running in anonymous mode. Anonymous mode does not give you any protections in single cluster either so nothing has changed there.